app/src/main/java/com/teamagam/gimelgimel/app/model/ViewsModels/messages/MessageDetailViewModel.java

```diff
@@ -35,6 +36,10 @@ private boolean isSelectedMessageOfType(@Message.MessageType String messageType)
         return mSelectedMessageModel.getSelected().getType().equals(messageType);
     }
 
+    public void drawMessageOnMap(MessagesDetailBaseGeoFragment.GeoMessageInterface drawMessageOnMapInterface) {
```

Comment:
about the VMMV pattern: we do not have entities model yet. 
I guessed that you will comment below about this topic :).
so just like you have said, I would not change it now but I totally agree with you about our future plans.